src_features/performPrivacyOperation/logic_performPrivacyOperation.c

```diff
@@ -0,0 +1,10 @@
+#include "shared_context.h"
+
+uint32_t set_result_perform_privacy_operation() {
+    uint32_t tx = 0, i;
+    for (i=0; i<32; i++) {
+        G_io_apdu_buffer[i] = tmpCtx.publicKeyContext.publicKey.W[32 - i];
+    }
+    tx = 32;
+    return tx;
```

Comment:
why put `tx` at 0 for the hard code at 32
The function will always return 32 I don't see the point

Reply:
noted